src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx

```diff
@@ -87,20 +88,19 @@ export const WalletMoreInfoCategory = ({
                 <ListIcon as={FeatureIcon}></ListIcon>
 
                 <Text px={1} lineHeight={1}>
-                  {feature.label}
+                  {feature.label}&nbsp;
```

Comment:
☹️ This is pesky... I'm still seeing this wrap. Made some adjustments locally but only getting it to work on Safari but not Brave.
In either case, I would stick with either the non-breaking space, or the margin (`ms="2"` below) but not both... I'm still not sure of a clean way to do this for all browsers.

Reply:
@Dharmik79 Just checking in... you still willing/able to help with this, or would you prefer someone to try and take it from here?

Reply:
@wackerow I think removing the ms="2" will be a better option.